app/controllers/announcements_controller.rb

```diff
@@ -34,4 +36,28 @@ def index
   def scheduled_announcements
     Announcement.scheduled.order("scheduled_publication asc")
   end
+
+  def is_english_locale?
```

Comment:
Unlike me to ever say something about comments (I've just read something, thinking of changing my ways). Anyway, might be worth a comment on why we're only redirecting english locales

Reply:
I've put a note about this in the commit message. Would you also like a comment in the code?